### ISSUE TITLE
JabRef 5.0: Main menus "File" & "Library" are confusing

### ISSUE THREAD

**Issue by mlep** [JabRef/jabref#5698]
Testing JabRef 5.0beta (JabRef 5.0.1-dev--2019-11-29----9043f8008), and after using JabRef 4.3.1, I find the main menu confusing, especially the menu File and the menu Library:
- under File, the main items are about "Library" (Open library, Save Library, etc.)
- under Library, there is an item "Library properties".

Suggestion:
- move the item "Library properties" to the menu File.
- rename the menu File into Library
- rename the menu Library into Content

Alternative:
- move the item "Library properties" to the menu File.
- in the menu file, replace the word 'library' with the word "file"

**Comment by tobiasdiez** [comment]
The logic was that everything reaching outside (open/save/import/export/...) is under "File", while everything specific to the currently open library is under "Library".

Is this really confusing? I find it quite intuitive.

**Comment by mlep** [comment]
I got the logic now. It makes sense.
To me, the consequence of this logic would be to adapt the name of the items, i.e. the word "library" should not appear in the menu File.

**Comment by AEgit** [comment]
I also do not see any problems with the current scheme. If you really wanted to change something, then maybe, as @mlep suggested, replace the word "Library" with "File" in the File menu, but to be honest, I don't think even that is necessary (it might actually cause more confusion).

**Comment by calixtus** [comment]
> The logic was that everything reaching outside (open/save/import/export/...) is under "File", while everything specific to the currently open library is under "Library".
> 
> Is this really confusing? I find it quite intuitive.

I think, it's intuitive for tech-people. For other people, who don't care about the tech details, not. But as almost everybody working with Bib(La)Tex, LaTeX and JabRef is at least a little bit some kind of nerd, it's probably just fine here.
No offense.

**Comment by mlep** [comment]
Just consider in the forums and mailing lists, the number of LaTeX users who confuse the GUI (TeXstudio, etc.) and the LaTeX engine. 
Nerds maybe, but not computer nerds!

Need another proof: they need a GUI. :wink:

**Comment by koppor** [comment]
Refs https://github.com/JabRef/jabref/issues/2095, which also provides pointers to other discussions (https://github.com/JabRef/jabref/pull/1988)

**Comment by mlep** [comment]
Something more about the consistency of the menus:

**1) Library vs Edit**
- in the menu "Library", there is the item "Delete entry". By default, the user is asked for confirmation of the deletion.
- in the menu "Edit", there is the item "Cut".  An entry can be cut, but no confirmation message is displayed at that time, nor when saving.

Basically, they do the same thing, "cut" being a bit more liberal.

Suggestion: remove the item "Delete entry". And move the item "Library -> New entry" to the "Edit" menu. In such a way:
- the menu "Edit" will be for managing/editing the individual entries
- the menu "Library" will be for the things concerning the whole library (preamble, strings, etc.)

**2) Quality and local files**
I would although suggest to move "Tools -> Search for unlinked local files" to the menu "Quality" since:
- the menu "Quality" already contains the field "Automatically set file links", a closely related item.
- the search for unlinked local files is about tidy up things, i.e. quality.


JabRef 5.0-beta.338--2020-01-13--b275b81
Linux 4.9.0-11-amd64 amd64 
Java 13.0.1

**Comment by AEgit** [comment]
Is "Delete entry" and "Cut" actually the same (irrespective of the confirmation dialog)?
For "Delete entry" I would expect the entry to just be deleted. For "Cut" I would expect that the entry is put into the clipboard/buffer and that it can be inserted elsewhere, e. g. be made part of a second library.

Furthermore, the additional confirmation dialogue for the "Delete entry" action seems a good idea. This way, a user should be less likely to accidentally delete a bunch of items.

**Comment by koppor** [comment]
Current state:

![grafik](https://user-images.githubusercontent.com/1366654/73555938-c7e47f00-444e-11ea-9edd-4b8f87ca1ac9.png)

![grafik](https://user-images.githubusercontent.com/1366654/73555960-ce72f680-444e-11ea-90d8-e1e0f1a948e4.png)

![grafik](https://user-images.githubusercontent.com/1366654/73555977-d59a0480-444e-11ea-8f57-cdfc4ecc449a.png)

## Context menu

![grafik](https://user-images.githubusercontent.com/1366654/73556038-f1050f80-444e-11ea-8621-dd4a3bf47649.png)

**Comment by koppor** [comment]
User experience is an important thing. We ran a master's thesis on JabRef's UI. Paper: http://ceur-ws.org/Vol-2339/paper10.pdf

We discussed much about "Library vs. Edit". Even we know something about UI design from the master's thesis, from our studies and from hearing, we agreed that much more energy should be spent on a "good" UI.

We currently have no time for doing a real user study and experiments.

Therefore, we close this issue without any further action in the next time. It will surely come up again - for instance, when working on a Microsoft Word integration.

Note to myself: Maybe we could experiment with Ribbons: https://www.pixelduke.com/fxribbon/
